GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -98,7 +98,9 @@ public CommitInfo()
                 this.InvokeAsync(() =>
                 {
                     UICommandsSource.UICommandsChanged += UICommandsSource_UICommandsChanged;
-                    ReloadCommitInfo();
+
+                    // call this event handler also now (necessary for "Contained in branches/tags")
+                    UICommandsSource_UICommandsChanged(null, null);
```

Comment:
Normally, we do not call the event handler directly but use a shared method.

Reply:
Yes, I have eliminated the separate handler.